images/linux/scripts/installers/boost.sh

```diff
@@ -7,16 +7,17 @@
 # Source the helpers for use with the script
 source $HELPER_SCRIPTS/document.sh
 
-TOOLSET_PATH="$INSTALLER_SCRIPT_FOLDER/toolcache.json"
-BOOST_LIB=/usr/local/share/boost
-BOOST_VERSIONS=$(cat $TOOLSET_PATH | jq -r 'to_entries[] | select(.key | match("boost")) | .value[] +".0"')
+TOOLSET_PATH="$INSTALLER_SCRIPT_FOLDER/toolset.json"
+BOOST_LIB="$AGENT_TOOLSDIRECTORY/boost"
+BOOST_VERSIONS=$(cat $TOOLSET_PATH | jq -r '.toolcache[] | select(.name | contains("boost")) | .versions[]')
+BOOST_ARCH=$(cat $TOOLSET_PATH | jq -r '.toolcache[] | select(.name | contains("boost")) | .arch')
 
 # Install Boost
 for BOOST_VERSION in ${BOOST_VERSIONS}
 do
     BOOST_SYMLINK_VER=$(echo "${BOOST_VERSION//[.]/_}")
     BOOST_ROOT_VERSION="BOOST_ROOT_$BOOST_SYMLINK_VER"
 
-    echo "$BOOST_ROOT_VERSION=$BOOST_LIB/$BOOST_VERSION" | tee -a /etc/environment
+    echo "$BOOST_ROOT_VERSION=$BOOST_LIB/$BOOST_VERSION/$BOOST_ARCH" | tee -a /etc/environment
     DocumentInstalledItem "Boost C++ Libraries $BOOST_VERSION"
```

Comment:
Please remove this line. Otherwise, Boost will shown twice in logs. Validate-Toolset already adds Boost to docs

Reply:
Removed it